src/changes/changes.xml

```diff
@@ -46,6 +46,9 @@
        <action issue="CONFIGURATION-805" type="update" dev="ggregory" due-to="Roman Zaynetdinov">
          [Javadoc] Specify that typed getList returns null for missing key #100.
        </action>
+       <action issue="CONFIGURATION-764" type="update" dev="ningzhangnj"  due-to="Ning Zhang">
+         Default date lookup can not work for some specific format
+       </action>
```

Comment:
You can leave `changes.xml` to be updated later by a committer, no need to include in this PR :+1: (it's also conflicting with `master` now, so removing it from this PR will fix that).